pkg/cmd/pulumi/util.go

```diff
@@ -71,6 +71,17 @@ func disableProviderPreview() bool {
 	return cmdutil.IsTruthy(os.Getenv("PULUMI_DISABLE_PROVIDER_PREVIEW"))
 }
 
+func disableResourceReferences() bool {
+	// TODO: Temporarily disabling resource ref support (https://github.com/pulumi/pulumi-kubernetes/issues/1405)
+
+	// Allow the resource reference feature to be disabled by explicitly setting an env var.
+	//	if v, ok := os.LookupEnv("PULUMI_DISABLE_RESOURCE_REFERENCES"); ok && cmdutil.IsTruthy(v) {
+	//		hasSupport = false
+	//	}
+
```

Comment:
We can remove the comment, close pulumi-kubernetes#1405, and uncomment the code as well, right?

Reply:
> We can remove the comment, close pulumi-kubernetes#1405, and uncomment the code as well, right?

I wasn't certain that was the plan. I don't think that we want to enable resource refs by default until we've landed all of the planned test coverage, do we?

Reply:
Sorry, I didn't mean enable by default, but don't we need some way to do the reverse (enable so we can test)? I assumed that logic would live in this block.

Reply:
> don't we need some way to do the reverse (enable so we can test)? I assumed that logic would live in this block.

Not for the purposes of these tests, which enable and disable support for resource refs by hitting the engine directly. For E2E/integration tests, though, we do need to restore the `PULUMI_ENABLE_RESOURCE_REFERENCES` envvar. I'll make that change.